api/users/views.py

```diff
@@ -17,13 +17,21 @@ class UserMixin(object):
     node_lookup_url_kwarg = 'user_id'
 
     def get_user(self, check_permissions=True):
+
+        key = self.kwargs[self.node_lookup_url_kwarg]
+        if key == 'me':
+            return self.request.user
+        elif check_permissions:
```

Comment:
This `elif` block seems to be a duplicate of the code that follows. I suspect maybe this is the result of a merge conflict?